Core/src/Surfaces/AnnulusBounds.cpp

```diff
@@ -178,158 +178,156 @@ bool Acts::AnnulusBounds::inside(const Vector2& lposition,
+
+  // covariance is given in STRIP PC
+  auto covStripPC = bcheck.covariance();
+  // calculate covariance in MODULE PC using jacobian from above
+  auto covModulePC = jacobianStripPCToModulePC * covStripPC *
+                     jacobianStripPCToModulePC.transpose();
+
+  // Mahalanobis distance uses inverse covariance as weights
+  auto weightStripPC = covStripPC.inverse();
+  auto weightModulePC = covModulePC.inverse();
+
+  double minDist = std::numeric_limits<double>::max();
+
+  Vector2 currentClosest;
+  double currentDist = 0;
+
+  // do projection in STRIP PC
+
+  // first: STRIP system. locpo is in STRIP PC already
+  currentClosest = closestOnSegment(m_inLeftStripPC, m_outLeftStripPC,
+                                    locpo_rotated, weightStripPC);
+  currentDist = squaredNorm(locpo_rotated - currentClosest, weightStripPC);
+  minDist = currentDist;
+
+  currentClosest = closestOnSegment(m_inRightStripPC, m_outRightStripPC,
+                                    locpo_rotated, weightStripPC);
+  currentDist = squaredNorm(locpo_rotated - currentClosest, weightStripPC);
+  if (currentDist < minDist) {
+    minDist = currentDist;
+  }
 
-    currentClosest = closestOnSegment(m_outLeftModulePC, m_outRightModulePC,
-                                      locpoModulePC, weightModulePC);
-    currentDist = squaredNorm(locpoModulePC - currentClosest, weightModulePC);
-    if (currentDist < minDist) {
-      minDist = currentDist;
-    }
+  // now: MODULE system. Need to transform locpo to MODULE PC
+  //  transform is STRIP PC -> STRIP XY -> MODULE XY -> MODULE PC
+  Vector2 locpoStripXY(
+      locpo_rotated[eBoundLoc0] * std::cos(locpo_rotated[eBoundLoc1]),
+      locpo_rotated[eBoundLoc0] * std::sin(locpo_rotated[eBoundLoc1]));
+  Vector2 locpoModulePC = stripXYToModulePC(locpoStripXY);
+
+  // now check edges in MODULE PC (inner and outer circle)
+  // assuming Mahalanobis distances are of same unit if covariance
+  // is correctly transformed
+  currentClosest = closestOnSegment(m_inLeftModulePC, m_inRightModulePC,
+                                    locpoModulePC, weightModulePC);
+  currentDist = squaredNorm(locpoModulePC - currentClosest, weightModulePC);
+  if (currentDist < minDist) {
+    minDist = currentDist;
+  }
 
-    // compare resulting Mahalanobis distance to configured
-    // "number of sigmas"
-    // we square it b/c we never took the square root of the distance
-    return minDist < bcheck.tolerance()[0] * bcheck.tolerance()[0];
+  currentClosest = closestOnSegment(m_outLeftModulePC, m_outRightModulePC,
+                                    locpoModulePC, weightModulePC);
+  currentDist = squaredNorm(locpoModulePC - currentClosest, weightModulePC);
+  if (currentDist < minDist) {
+    minDist = currentDist;
   }
+
+  // compare resulting Mahalanobis distance to configured
+  // "number of sigmas"
+  // we square it b/c we never took the square root of the distance
```

Comment:
```suggestion
  // Compare resulting Mahalanobis distance. To configured the "number of sigmas"
  // we square it, because we never took the square root of the distance.
```
This might require formatting.